functional_tests/artifacts/link-model.yea

```diff
@@ -0,0 +1,9 @@
+id: 0.artifacts.link-model
+plugin:
+  - wandb
+assert:
+  - :wandb:runs_len: 1
+  - :wandb:runs[0][config]: {}
+  - :wandb:runs[0][exitcode]: 0
+  - :wandb:artifacts[my-model][type]: model
```

Comment:
nit: are there other fields we can test here?

Reply:
portfolio-links (line 9) should cover the test case; I can also add aliases but not sure how yea's assertion works with lists/arrays. Best thing for aliases would be set equality -- I don't think yea supports that yet, right?